### PR TITLE
virtcontainers: Remove duplicated assert msg in utils test code

### PR DESCRIPTION
**Pull request by Kvasscn**
The last argument 'msg' in assert.Errorf() and assert.NoErrorf()
seems unnecessary, so remove it.

Fixes: #3714

Signed-off-by: zhanghj <zhanghj.lc@inspur.com>

### CODE REVIEW

File: src/runtime/virtcontainers/utils/utils_test.go [hunk 1]
```diff
@@ -297,9 +297,9 @@ func TestBuildSocketPath(t *testing.T) {
 		msg := fmt.Sprintf("test[%d]: %+v", i, d)
 
 		if d.valid {
-			assert.NoErrorf(err, "test %d, data %+v", i, d, msg)
```
Comment (liubin) [inline src/runtime/virtcontainers/utils/utils_test.go:300]: Nit: how about deleting `"test %d, data %+v", i, d,` and keep `msg`?

Comment (Kvasscn) [reply src/runtime/virtcontainers/utils/utils_test.go]: ok, thanks.

Comment (Kvasscn) [reply src/runtime/virtcontainers/utils/utils_test.go]: The previous PR is closed by  misoperation, please review again. @liubin

Comment (Kvasscn) [reply src/runtime/virtcontainers/utils/utils_test.go]: resubmit the pull request at  https://github.com/kata-containers/kata-containers/pull/3718

**Pull request closed without merging**